tests/Directory.Build.targets

```diff
@@ -0,0 +1,16 @@
+<Project>
+  <Import Project="..\Directory.Build.targets" />
+
+  <ItemGroup>
+    <None Include="$(RepositoryEngineeringDir)testing\.runsettings" CopyToOutputDirectory="PreserveNewest" Condition="'$(DeployRunSettingsFile)' == 'true'" />
+  </ItemGroup>
+
+  <Target Name="ZipTestArchive" AfterTargets="Build" Condition="'$(ArchiveTests)' == 'true' and '$(RunTestOnHelix)' == 'true'">
+    <Error Condition="'$(TestArchiveTestsDir)' == ''" Text="TestArchiveTestsDir property to archive the test folder must be set." />
+
+    <MakeDir Directories="$(TestArchiveTestsDir)" />
+    <ZipDirectory SourceDirectory="$(OutDir)"
```

Comment:
Interesting, I would have imagined that zipping up build output would be standard and provided by the Helix SDK.

Reply:
Helix allows you to point to a directory which it can then upload as a zip file. But doing it this way gives us a way to also indicate the tests meant for helix. This is the same technique followed in `dotnet/runtime`, though we don't have to do the same thing.